torch/quantization/fx/quantize.py

```diff
@@ -213,17 +216,11 @@ def convert(self, quantizer, node, load_arg, debug=False):
             else:
                 self.conv.activation_post_process = quantizer.activation_post_process_map[node.name]
             # 2. select quantized class
-            if type(self.conv) in [torch.nn.Conv2d,
-                                   torch.nn.qat.Conv2d,
-                                   torch.nn.intrinsic.qat.ConvBn2d]:
-                qconv = torch.nn.quantized.Conv2d
-            elif type(self.conv) in [torch.nn.intrinsic.ConvReLU2d,
-                                     torch.nn.intrinsic.qat.ConvReLU2d,
-                                     torch.nn.intrinsic.qat.ConvBnReLU2d]:
-                qconv = torch.nn.intrinsic.quantized.ConvReLU2d
-            else:
-                raise Exception("unhandled conv type:", type(self.conv))
-            quantized = qconv.from_float(self.conv)
+            # TODO: make the mapping configurable?
+            assert type(self.conv) in DEFAULT_MODULE_MAPPING, \
```

Comment:
This is a looser check than the previous check which was checking for specific module classes. Is this sufficient?

Reply:
yeah it is equivalent, the check on the left side can be represented by the mapping.